demo/admin/src/common/MasterMenu.tsx

```diff
@@ -24,10 +33,12 @@ const MasterMenu: React.FC = () => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [location]);
 
-    const sectionScopeTitle = `${scope?.domain.charAt(0).toUpperCase() + scope?.domain.slice(1)} - ${scope?.language?.toUpperCase()}`;
+    const sectionScopeTitle = `${scope?.domain.charAt(0).toUpperCase() + scope?.domain.slice(1)} - ${
+        values.language[0].label ?? scope?.language?.toUpperCase()
+    }`;
```

Comment:
Why do you need to change this?

Reply:
I actually don't know anymore 😅  But it was a dedicated commit (50a0794c5b34b28d63cde125a8bf16c0ab6714eb). Maybe @thomasdax98 can remember it.

Reply:
BTW, the `values.language[0].label ?? scope?.language?.toUpperCase()` logic is incorrect since it always takes the first language, not the language of the current scope. I'd say we revert it.

Reply:
The idea was displaying the label instead of the scope value ("English" instead of "EN")